actions/transfer-case.vbs

```diff
@@ -139,8 +140,12 @@ DO
 
     '----------Checks that the worker or agency is valid---------- 'must find user information before transferring to account for privileged cases.
     IF transfer_to_worker = "X127CCL" THEN script_end_procedure("This case is will be transferred via an automated script after being closed for 4 months, the script will now end.")
-
+    IF worker_number = transfer_to_worker THEN          'If they match, cancel the transfer and save the information about the 'failure'
+        end_msg = "This case is already in the requested worker's number."
```

Comment:
Line 144 can be removed since the message is already in the script end procedure on line 145